assert.go

```diff
@@ -79,6 +79,12 @@ func Equal(t testingT, got, want interface{}, opts ...cmp.Option) bool {
 	return assertEqual(t, getArg(1), got, want, opts)
 }
 
+// NotEqual asserts that got and want are assertNotEqual.
```

Comment:
```suggestion
// NotEqual asserts that got and want are not equal.
```
(while you're in there, maybe fix the docs on `Equal`? my guess is I renamed a helper function from `equal` to `assertEqual` and accidentally changed the word in the doc comment as well.